test/unit/style-spec/fixture/text-field-format.input.json

```diff
@@ -0,0 +1,45 @@
+{
+  "version": 8,
+  "sources": {
+    "point": {
+      "type": "geojson",
+      "data": {
+        "type": "FeatureCollection",
+        "features": [
+          {
+            "type": "Feature",
+            "properties": {
+              "name": "Napoli",
+              "name_en": "Naples",
+              "name_color": "blue"
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [ 0, 0 ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "glyphs": "local://glyphs/{fontstack}/{range}.pbf",
+  "layers": [
+    {
+      "id": "text",
```

Comment:
nit: Name the layer `id` more descriptively a la [`text-font.input.json`](https://github.com/mapbox/mapbox-gl-js/blob/0934e4e1f2a71e0cd492406cc62f81cfaea5b6ef/test/unit/style-spec/fixture/text-font.input.json#L32)

You can also minimize the test case by removing `text-font` and `text-size`.